parm/soca/obsproc/obsproc_config.yaml

```diff
@@ -12,14 +12,30 @@ observations:
     output file: sss_smos.ioda.nc
     obsproc regex: SM_OPER_MIR_OSUDP2_????????T??????_????????T??????_700_001_1.nc
 - obs space:
-    name: adt_3b_egm2008
+    name: adt_rads_all
     obsproc subdir: ADT
-    obsproc regex: rads_adt_3b_???????.nc
+    obsproc regex: rads_adt_??_???????.nc
     provider: RADS
-    output file: adt_all.nc4
+    output file: adt_rads_all.nc4
 - obs space:
     name: icec_amsr2_north
     provider: AMSR2
     obsproc subdir: icec
-    output file: icec_amsr2_north_1.ioda.nc
+    output file: icec_amsr2_north.ioda.nc
     obsproc regex: AMSR2-SEAICE-NH_v2r2_GW1_s???????????????_e???????????????_c???????????????.nc
+- obs space:
+    name: icec_amsr2_south
+    provider: AMSR2
+    obsproc subdir: icec
+    output file: icec_amsr2_south.ioda.nc
+    obsproc regex: AMSR2-SEAICE-NH_v2r2_GW1_s???????????????_e???????????????_c???????????????.nc
```

Comment:
Should be:
obsproc regex: AMSR2-SEAICE-SH_v2r2_GW1_s???????????????_e???????????????_c???????????????.nc